app/src/main/java/edu/uc/kovaciad/slicetracker/dao/IMaterialDAO.kt

```diff
@@ -0,0 +1,5 @@
+package edu.uc.kovaciad.slicetracker.dao
```

Comment:
IMaterialDAO_Impl, IModelDAO_Impl, IPrinterDAO_Impl and IBrandDAO_Impl referenced files that did not exist. I added a framework to fix the reference error but did not have enough info about what you would need from them to flesh them out. 

As a side note several methods were missing references from the data classes as well. Since I could not fix those however, I have not changed them.

Reply:
Where are these referenced?  I don't see them anywhere.